### ISSUE TITLE
Noto Serif's "f with hook"(ƒ) is different from Sans

### ISSUE THREAD

**Issue by NDPtype** [notofonts/noto-fonts#2399]
When I search for Noto fonts on Google Fonts, I found that Noto Serif's "f with hook" looks different from Noto Sans.
![noto inconsistent](https://user-images.githubusercontent.com/105497069/186071173-fe564cb4-501a-45c8-b0ef-eeeb9ee6ae19.png)
Noto Serif should use the same structure as Noto Sans, so that it would be consistent.

**Comment by fitojb** [comment]
Indeed, the letter should be upright as it is used in the orthography of a living language (Ewe) and not as a currency, but I couldn’t help but notice that the terminal of the Sans variant is cut in a quite ugly angle.
